.gitignore

```
@@ -30,3 +30,6 @@
 
 # ENV file
 .env
+
+# .gitignore
+node_modules/
```

Please also add the routes for your controller in the routes.rb file. you can follow the format bellow:
```suggestion

namespace :api do
    namespace :v1 do
      resources :reservations, only: [:index :create]
    end
  end
```